### ISSUE TITLE
SCSS Flags removed?

### ISSUE THREAD

**Issue by norfali** [assetsjs/postcss-assets#43]
Any SCSS files containing a `!default` or `!global`, postcss-assets seem to remove them. e.g.
```
$var: 100 !default;
$foo: "hello" !global;
$something: "bye" !important;
```

converts to:
```
$var: 100 ;
$foo: "hello" ;
$something: "bye" !important;
```

Simple gulp function as follows:
```
return gulp.src(path.join(contextObject.srcDir, 'style', '**', '*.{scss,css}'))
        .pipe($.postcss([require('postcss-assets')({
            loadPaths: contextObject.cssAssetPaths
        })], { syntax: require('postcss-scss') }))
...
}
```

The task runs fine and inline of assets work but I want to retain these flags so I am unsure whether this is a postcss-assets bug or there is an option somewhere.

Any ideas? Many thanks!

**Comment by borodean** [comment]
Fixed in 4.0.0. Could you verify?

**Comment by norfali** [comment]
Thanks - I can verify it works now and bug is fixed.
